src/main/java/seedu/address/model/group/Group.java

```diff
@@ -11,6 +11,10 @@ public class Group {
 
     public static final String MESSAGE_CONSTRAINTS = "Groups names should be in correct format with 2 digit number. "
         + "EX) g/TUT04, g/LAB10, g/REC09>. Link should be a valid Telegram invite link.";
+
+    public static final String MESSAGE_CONSTRAINTS_KEYWORD = "Groups names should be in correct format "
+            + "with 2 digit number. "
+            + "EX) TUT04, LAB10, REC09>.";
```

Comment:
Is there an extra ">" at the end of the string? Also, just a nit but perhaps using "E.g." may be more intuitive than "EX)" for examples?